psychopy/tests/test_monitors/test_monitors.py

```diff
@@ -0,0 +1,30 @@
+from psychopy.monitors.calibTools import Monitor
+import pytest
+
+
+@pytest.mark.monitors
+class TestMonitorCalibration(object):
+    def setup(self):
+        pass
+
+    def teardown(self):
+        pass
+
+    def test_reload_monitor(self):
```

Comment:
For intelligibility of the tests, it is often best to use simple functions instead of `pytest` classes for simple test cases like this one. This suggestion of mine is a _can_, not a _must_. :)